lib/main.dart

```diff
@@ -22,7 +23,7 @@ class MyApp extends StatelessWidget {
       GoRoute(
         path: routePathRootScreen,
         builder: (BuildContext context, GoRouterState state) =>
-            const HomeScreen(),
+            const LoginScreen(),
```

Comment:
We can shorten this code by omitting the types.

```suggestion
        builder: (context, state) => const LoginScreen(),
```

Or even omit the parameters' name as we don't use these values:

```suggestion
        builder: (_, __) => const LoginScreen(),
        // change `context` to 1 underscore
        // change `state` to 2 underscores
```